manual/src/html_processing/src/process_manual.ml

```diff
@@ -111,8 +150,9 @@ let load_html file =
 
   (* For the main index file, we do a few adjustments *)
   let html = if file = "index.html"
-    then Re.Str.(global_replace (regexp "Part \\([I|V]+\\)<br>")
-                   "<span>\\1. </span>" html)
+    then Re.Str.(global_replace
+                   (regexp ("Part\u{2004}\\([I|V]+\\)<br>"))
+                   {|<span class="number">\1.</span>|} html)
```

Comment:
Small suggestion: we could remove the "\n" after the `<br>` tag to avoid introducing a space.

Reply:
good idea, done.